front-office/checkout.md

```diff
@@ -33,6 +33,15 @@ If the store has only one payment method available, this payment method should b
 
 ## Final summary
 
+
+If the merchant has enabled the 'Final summary' in the Shop Parameters > Order Settings section of the back office, a panel summarizing the order is displayed below the paiment block.
+It displays the address of shipping and invoice, the shipping method ,and the product list of the cart.
```

Comment:
Maybe add the info "and recap of product and shipping price, discount and total" ? (well, something like that but in good english)